pkg/client/unversioned/helper.go

```diff
@@ -672,3 +709,18 @@ func TimeoutFromListOptions(options api.ListOptions) time.Duration {
 	}
 	return 0
 }
+
+func stripAPIFromPrefix(base string) string {
+	// normalize the base
```

Comment:
shouldn't need to worry about trailing / since paths are Join'd and part of being Join's is being Clean'ed - and that strips a trailing /